AssemblyInfo.cs

```diff
@@ -43,6 +43,8 @@
 [assembly: InternalsVisibleTo("Microsoft.Exchange.Test.Migration.MultiMailboxSearch,PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
 [assembly: InternalsVisibleTo("Internal.Exchange.EDiscovery.MailboxSearch.FunctionalTest, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
 [assembly: InternalsVisibleTo("Internal.Exchange.EDiscovery.MailboxSearch.UnitTest, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
+[assembly: InternalsVisibleTo("Microsoft.Exchange.WebServices.Tests")]
```

Comment:
General comment: did you know there are some EWS forks that support .Net Core like https://github.com/sherlock1982/ews-managed-api or https://github.com/OfficeDevUnofficial/ews-managed-api? For now I think make sense to keep using our fork but if we will have more issues we could consider switching to those or maybe even to Graph API (since EWS is discontinued by Microsoft).

Reply:
Yes, I know. In fact, first thing I tried to do is to switch to one of those forks. Unfortunately our fork already contains some new abstractions used in Next, so it is not easy to switch. Also - they contain their own issues and bugs. I would like to switch to something else in the future if we can justify the effort. For now we have maybe 1 issue in 6 months, so does not seem like a high priority.

Reply:
Yeap, I think the same. Good that you are also aware of those forks.